x-pack/plugins/apm/public/components/app/TransactionOverview/__jest__/TransactionOverview.test.tsx

```diff
@@ -85,7 +83,7 @@ describe('TransactionOverview', () => {
   const FILTER_BY_TYPE_LABEL = 'Transaction type';
 
   describe('when transactionType is selected and multiple transaction types are given', () => {
-    it('should render dropdown with transaction types', () => {
+    it('should render radio group with transaction types', () => {
```

Comment:
These read better without "should":
```suggestion
    it('renders a radio group with transaction types', () => {
```